src/requestCertificate.jsx

```diff
@@ -99,31 +98,41 @@ const NicknameRow = ({ onValueChanged, dialogValues }) => {
 };
 
 const CertFileRow = ({ onValueChanged, dialogValues }) => {
+    /* FileAutoComplete does not accept custom className and without it the Select gets shrinked
+     * because of ct-form. Let's just have this workaround and fix this properly by *not* using ct-form
+     */
     return (
         <>
             <label className="control-label" htmlFor="cert-file">
                 {_("Certificate path")}
             </label>
-            <FileAutoComplete id="cert-file"
-                superuser="try"
-                placeholder={_("Path to store the certificate")}
-                onChange={value => onValueChanged("certFile", value)}
-                fileExists={false} />
+            <span className="ct-form-stretch">
+                <FileAutoComplete id="cert-file"
+                    isOptionCreatable
+                    superuser="try"
+                    placeholder={_("Path to store the certificate")}
+                    onChange={value => onValueChanged("certFile", value)} />
+            </span>
         </>
     );
 };
 
 const KeyFileRow = ({ onValueChanged, dialogValues }) => {
+    /* FileAutoComplete does not accept custom className and without it the Select gets shrinked
+     * because of ct-form. Let's just have this workaround and fix this properly by *not* using ct-form
+     */
     return (
         <>
             <label className="control-label" htmlFor="key-file">
                 {_("Key path")}
             </label>
-            <FileAutoComplete id="key-file"
-                superuser="try"
-                placeholder={_("Path to store the generated key or to an existing key")}
-                onChange={value => onValueChanged("keyFile", value)}
-                fileExists={false} />
```

Comment:
this is where it went missing. This parameter basically says "Filepath which user input's doesn't have to be present on a filesystem"

Reply:
So perhaps it's best for me to introduce that functionality to a pkg/lib/cockpit-components-file-autocomplete.jsx in a cockpit repo

Reply:
See comment above.

Reply:
Ahhh, now I see the PR where you requested my review.... Sorry it went by my radar

Reply:
no problem it that was only 1h ago or so.